config/locales/en.yml

```diff
@@ -1276,7 +1276,7 @@ en:
         form_label: Grant Special User privileges
         form_placeholder: Username
         confirm_add_special_user: "Add Special User"
-        confirm_remove_special_user: "Remove Special User privileges for"
+        _user: "Remove Special User privileges for"
```

Comment:
What is this for?

Reply:
Looks like it is unused as well @ragesoss

Reply:
Oh, I guess it was a typo and you intended to remove the whole line?

Reply:
> Oh, I guess it was a typo and you intended to remove the whole line?

Exactly!
Removed it completely now